src/hip_hcc_internal.h

```diff
@@ -802,6 +802,9 @@ class ihipDevice_t {
 
     // TODO - report this through device properties, base on HCC API call.
     int _isLargeBar;
+   
+    // Node id reported by kfd for this device
+    uint32_t internal_node_id;
```

Comment:
Rename to _node_id or _driver_node_id

Reply:
renamed to _driver_node_id. updated the PR.